pkg/deploy-manager/config.go

```diff
@@ -27,8 +27,15 @@ const DefaultStorageClusterName = "test-storagecluster"
 // DefaultStorageClassRBD is the name of the ceph rbd storage class the test suite installs
 const DefaultStorageClassRBD = DefaultStorageClusterName + "-ceph-rbd"
 
-// MinOSDsCount represents the minimum number of OSDs required for this testsuite to run.
-const MinOSDsCount = 3
+const minOSDsCount = 3
+const minOSDsCountArbiter = 4
+
+func (t *DeployManager) getMinOSDsCount() int {
+	if t.IsArbiterEnabled() {
+		return minOSDsCountArbiter
```

Comment:
This is the only change I see related to OSDs. Will this cause the `storageClassDeviceSet.Count` to be `4`? I believe the node affinity, topology spread constraints, etc would be the same in a stretch cluster as other non-arbiter clusters, as long as the topology spread constraints are reliable to spread the ODSs evenly first across the two zones, then across the hosts within zones. For example, say we have 4 worker nodes in each of the two non-arbiter zones, giving us 8 worker nodes where OSDs could be started. We need to ensure two OSDs end up in each zone, rather than all four OSDs potentially in the same zone. Do the OCS topology spread constraints handle that?

Reply:
Yes. The above scenario is covered; added in the `placement` related commit.